src/transformers/models/cohere/modeling_cohere.py

```diff
@@ -1071,12 +1078,19 @@ def forward(
         )
 
         hidden_states = outputs[0]
-        logits = self.lm_head(hidden_states)
+        if labels is None and not is_torchdynamo_compiling():
+            logger.warning_once(
+                "Starting from v4.45, the `logits` model output will have the same type as the model (except at train time, where it will always be FP32)"
```

Comment:
```suggestion
                "Starting from v4.46, the `logits` model output will have the same type as the model (except at train time, where it will always be FP32)"
```

Reply:
should be changed everywhere as next version is 4.45!

Reply:
Done but the `examples_tensorflow` test started to fail again on the CI 🥲